raft/raft.go

```diff
@@ -67,6 +67,8 @@ const (
 	campaignTransfer CampaignType = "CampaignTransfer"
 )
 
+var ErrProposalDropped = errors.New("raft proposal dropped")
```

Comment:
doc string on public error?

Reply:
// ErrProposalDropped is returned by the Step function when the passed in proposal is dropped.